projects/tuist/fixtures/app_with_plugins/Tuist/ProjectDescriptionHelpers/Project+Templates.swift

```diff
@@ -1,8 +1,14 @@
+import ExampleTuistPlugin
+import LocalPlugin
 import ProjectDescription
 
 extension Project {
     /// Helper function to create the Project for this ExampleApp
     public static func app(name: String, platform: Platform, additionalTargets _: [String]) -> Project {
+        // Note: Testing importing of plugins in local helpers
+        _ = LocalHelper(name: "local")
+        _ = RemoteHelper(name: "remote")
+
         let mainTarget = Target(
             name: name,
             platform: platform,
```

Comment:
Do you think there is a way to test this in CI? 🤔

Reply:
We probably still want this here so that the fixture is more robust, but I can see about adding a unit test asserting plugins are linked correctly.

Reply:
Added some tests 🚀